forge/db/models/Project.js

```diff
@@ -10,7 +10,7 @@ module.exports = {
     name: 'Project',
     schema: {
         id: { type: DataTypes.UUID, primaryKey: true, defaultValue: DataTypes.UUIDV4 },
-        name: { type: DataTypes.STRING, allowNull: false },
+        name: { type: DataTypes.STRING, allowNull: false, unique: true },
```

Comment:
```suggestion
        name: { type: DataTypes.STRING, allowNull: false },
```

I think it will be cleaner to *not* add this constraint with this PR - given we're taking a code-first approach to policing it at the moment.

Once we have figured out the migration issue with sqlite, we can add it to the model and have the corresponding migration ready to go - rather than have some databases with it and some databases without it.

Reply:
I've removed it and the migration from the PR